injective/assetlist.json

```diff
@@ -484,6 +484,131 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/astro.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/astro.svg"
       }
+    },
+    {
+      "description": "A receipt token for lent INJ issued by the Neptune Protocol.",
+      "denom_units": [
+        {
+          "denom": "inj1rmzufd7h09sqfrre5dtvu5d09ta7c0t4jzkr2f",
+          "exponent": 0
+        },
+        {
+          "denom": "nINJ",
+          "exponent": 18
+        }
+      ],
+      "base": "inj1rmzufd7h09sqfrre5dtvu5d09ta7c0t4jzkr2f",
+      "name": "Neptune Receipt INJ",
+      "display": "nINJ",
+      "symbol": "nINJ",
+      "logo_URIs": {
```

Comment:
Please remove logo_URIs
(may merge [this PR](https://github.com/cryptechdev/cosmos-chain-registry/pull/1))

Reply:
merged change to logo_URIs